tests/conftest.py

```diff
@@ -74,7 +74,7 @@ def is_equal(a, b, tolerance=1e-6):
         if a.dtype.kind in "UOS":  # U = unicode, O = objects, S = string
             return np.array_equal(a, b)
         else:
-            b = np.where(b is None, np.nan, b)  # Replace None with np.nan
+            b = np.where(b == None, np.nan, b)  # Replace None with np.nan
```

Comment:
I think that we should keep b is None. PyCharm is suggesting me that it is better